Distribution/Server/Framework/MemSize.hs

```diff
@@ -145,9 +163,15 @@ instance (MemSize a, MemSize b) => MemSize (Either a b) where
 instance (MemSize a, MemSize b) => MemSize (Map a b) where
   memSize m = sum [ 6 + memSize k + memSize v | (k,v) <- Map.toList m ]
 
+instance (MemSize a, MemSize b) => MemSize (Bimap a b) where
+  memSize m = sum [ 6 + memSize k + memSize v | (k,v) <- Bimap.toList m ]
+
 instance MemSize a => MemSize (IntMap a) where
   memSize m = sum [ 8 + memSize v | v <- IntMap.elems m ]
 
+instance MemSize Graph where
+  memSize m = sum [ 6 + memSize v | v <- Gr.edges m ]
```

Comment:
I don't know the right instance offhand, but feel free to go for it... (shouldn't be a blocker either way)